src/stratigraphy/data_extractor/data_extractor.py

```diff
@@ -105,7 +113,15 @@ def find_feature_key(self, lines: list[TextLine], allowed_error_rate: float = 0.
             for line in lines:
                 match = pattern.search(line.text)
                 if match:
-                    matches.add(line)
+                    # Make sure the key is not in the false positive list
+                    is_fp_key = False
+                    for fp_key in self.feature_fp_keys:
+                        if fp_key in line.text:
+                            is_fp_key = True
+                            break
+
+                    if not is_fp_key:
```

Comment:
Is this the same as 
```
if all(not fp_key in line.text for fp_key in self.feature_fp_keys):
```

Reply:
I used this experession as it seemed a bit easier to read for me: 

`if not any(fp_key in line.text for fp_key in self.feature_fp_keys):`